File: test2.py
import unittest

class TestMathMethods(unittest.TestCase):

    def test_add(self):
        self.assertEqual(1 + 1, 2)

    def test_subtract(self):
        self.assertEqual(3 - 1, 2)

if __name__ == '__main__':
    unittest.main()

File: Test2.java
import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class TestMathMethods {

    @Test
    public void testAdd() {
        int a = 1;
        int b = 1;
        int result = a + b;
        assertEquals(2, result);
    }

    @Test
    public void testSubtract() {
        int a = 3;
        int b = 1;
        int result = a - b;
        assertEquals(2, result);
    }
}
